### ISSUE TITLE
make background parallax

### ISSUE THREAD

**Issue by cyberpunkcoder** [cyberpunkcoder/reaction-control#2]

**Comment by tauraamui** [comment]
Does this mean make it scroll?

**Comment by cyberpunkcoder** [comment]
> Does this mean make it scroll?

No, actually the background already scrolls infinitely, and it only loads where it is visible. Parallax is where the stars move at a different rate as the nebula in the background.

Basically it will require splitting the background.png into two files, one with the stars and one with the nebula and move them at different rates of speed where the stars are moving past at the speed of the player to show movement and the "deep" background nebula moves more slowly to give a feeling of "depth".

**Comment by cyberpunkcoder** [comment]
This is just to label that the branch "feature-2" will be the implementation of this feature, I'm not currently working on this feature but you're welcome too if it interests you, if not feel free to work on something more appealing to you!

**Comment by tauraamui** [comment]
Ah yeah, I've done the same thing before, you offset it's position with the distance that you want. I'll maybe have a look :)
